plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -105,7 +105,7 @@ func NewQuotaEvaluator(quotaAccessor QuotaAccessor, registry quota.Registry, loc
 
 		registry: registry,
 
-		queue:      workqueue.New(),
+		queue:      workqueue.NewNamed("admission_quota_controller"),
```

Comment:
> I realize this is pkg/admission and not pkg/controller, but would you want to do admission_controller_resourcequota?

This drives the quota admission plugin.  If we have more like this, I think we'd want the top level partitioning to be `<area>_<name in that area>_<component of that name>`, so that all quota admission bits sort together instead of having all admission plugins that also have a controller sorting together.